### PR TITLE
 UI Defects

### PR DESCRIPTION
**Pull request by swati-sh**
Requirements

* If asset data was empty , proper message is not being displayed.
* Sort icon was not being displayed.
* While filtering data based on service name or domain  on click of enter, filter row is getting hidden.
* Css fix
 
Description of the Change
  Included  condition for asset data and sort icon. 

Benefits
 Now row is not being hidden and message is displayed.

### CODE REVIEW

File: core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.ts [hunk 1]
```diff
@@ -197,7 +197,7 @@ export class EnvAssetsSectionComponent implements OnInit {
     this.subscription = this.http.get(this.relativeUrl, payload, this.service.id).subscribe(
       (response) => {
 
-        if((response.data == undefined) || (response.data.length == 0)){
+        if((response.data == undefined) || (response.data.length == 0) || (response.data.count == 0)){
```
Comment (supritat) [inline core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.ts]: You can remove (response.data.length == 0) condition.

File: core/jazz_ui/src/app/pages/services-list/services-list.component.scss [hunk 2]
```diff
@@ -8,6 +8,20 @@
 /deep/ .table-wrap.fixed {
   position: fixed;
 }
+/deep/.service-list-table
+ .column:nth-child(3) 
+ .input-wrapper {
+    margin-left: 0 !important;
+}
+/deep/ .service-list-table
```
Comment (pgobin) [inline core/jazz_ui/src/app/pages/services-list/services-list.component.scss]: Avoid use of deep where currently possible.

File: core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.ts (hunk 1 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.ts]: Removed

File: core/jazz_ui/src/app/pages/services-list/services-list.component.scss (hunk 2 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/pages/services-list/services-list.component.scss]: ok

File: core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.ts [hunk 3]
```diff
@@ -197,7 +197,7 @@ export class EnvAssetsSectionComponent implements OnInit {
     this.subscription = this.http.get(this.relativeUrl, payload, this.service.id).subscribe(
       (response) => {
 
-        if((response.data == undefined) || (response.data.length == 0)){
+        if((response.data == undefined) || (response.data.count == 0)){
```
Comment (pgobin) [inline core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.ts]: Nit: prefer strict equality operator where appropriate.

**Pull request merged**